cl_firesim/simulation.cfg

```diff
@@ -1,2 +1,7 @@
 # Configuration settings used only for FPGA-level simulation
 # This is the hw_emu (hardware emulation) target of v++
+[advanced]
+param=compiler.userPreSysLinkOverlayTcl=../test.tcl
```

Comment:
Please use compiler.hw_emu.post_sim_settings=<fullpath>/test.tcl

Reply:
`v++` is called in a generated directory beneath this one though. Looking through the log, it seems like Vivado is properly applying these scripts.

Reply:
These scripts only apply to the user region. Not to the full project